.github/workflows/ci.yml

```diff
@@ -13,8 +13,7 @@ env:
   # This should be limited to packages that are intended for publishing.
   RUST_MIN_VER_PKGS: "-p kurbo"
   # List of features that depend on the standard library and will be excluded from no_std checks.
-  FEATURES_DEPENDING_ON_STD: "std,default,schemars"
-
+  FEATURES_DEPENDING_ON_STD: "std,default,schemars,euclid"
```

Comment:
euclid has no_std support. Can we turn off its default features, or do we need something from std?